iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -67,11 +67,16 @@ PortManager::PortManager(RouDiMemoryInterface* roudiMemoryInterface) noexcept
     registryPortOptions.nodeName = iox::NodeName_t("Service Registry");
     registryPortOptions.offerOnCreate = true;
 
-    m_serviceRegistryPublisherPortData = acquireInternalPublisherPortData(
+    // we cannot /fully) perform discovery without this port
```

Comment:
This is needed, otherwise we get the warning that the registry cannot be sent at startup all the time.